vocabularies/Org.OData.Validation.V1.xml

```diff
@@ -144,11 +144,11 @@ even though they are defined in an `Orders` entity set.</String>
           String="Dynamic properties added to the annotated open structured type are restricted to the listed types" />
       </Term>
 
-      <Term Name="DerivedTypeConstraint" Type="Collection(Core.QualifiedTypeName)" Nullable="false" AppliesTo="Property TypeDefinition">
+      <Term Name="DerivedTypeConstraint" Type="Collection(Core.QualifiedTypeName)" Nullable="false" AppliesTo="EntitySet Singleton NavigationProperty Property TypeDefinition Action Function Parameter">
         <Annotation Term="Core.Description"
-          String="Values are restricted to types derived from the declared type and listed in this collection" />
+          String="Values are restricted to types that are both identical to or derived from the declared type and a type listed in this collection" />

```

Comment:
Maybe a "."  at the end of string value of "Core.Description".

Reply:
Thanks!